### PR TITLE
feat: new page match list

### PR DESCRIPTION
**Pull request by urielPalomares**
#25

### CODE REVIEW

File: pages/matchs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          src={'/images/liverpool-fc-logo.png'}
+          alt='Local team logo'
+          width={150}
+          height={180}
+          className={styles.logoShadow}
+        ></Image>
+        <div className={styles.nameTeam}>
+          <h2>Liverpool</h2>
+        </div>
+      </div>
```
Comment (Andres2D) [inline pages/matchs/index.tsx:32]: This could be a component.

File: pages/matchs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+const Menu: NextPage = () => {
+
+  const { data: session } = useSession();
+  const router = useRouter();
+
+  if(!session) {
+    <p>Loading</p>
+  }
+
+  return (
```
Comment (Andres2D) [inline pages/matchs/index.tsx:19]: The component should be clean, all the layout must be created in the components.
Check the profile page.

**Review by Andres2D — CHANGES_REQUESTED** [review]

File: pages/matchs/matchs.module.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
+.container {
+  align-items: center;
+  justify-content: space-evenly;
+  margin: 100px auto;
+  width: 35%;
```
Comment (Andres2D) [inline pages/matchs/matchs.module.css:5]: Increase the width.

File: pages/matchs/index.tsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      </div>
+
+      <div className={styles.options}>
+        <div className={styles.option}>
+        <SettingsIcon w={7} h={7} color="gray.600" />
+        </div>
+        <div className={styles.option}>
+        <DeleteIcon w={7} h={7} color="red.600" />
+        </div>
+      </div>
```
Comment (Andres2D) [inline pages/matchs/index.tsx:68]: Use a button with icon component: lilke [this](https://chakra-ui.com/docs/components/button/usage#button-with-icon)

File: pages/matchs/matchs.module.css [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+.footer {
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  left: 40%;
+  bottom: 5%;
+  width: 20%;
+  color: white;
+  text-align: center;
+}
```
Comment (Andres2D) [inline pages/matchs/matchs.module.css:50]: The button should be fixed, just leave it in the bottom of the other components.

File: pages/matchs/matchs.module.css [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+.option {
+  background-color: white;
+  border-radius: 10px;
+  align-items: center;
+  text-align: center;
+  margin-top: 15%;
+  padding: 0.3rem;
+}
```
Comment (Andres2D) [inline pages/matchs/matchs.module.css:29]: Fix the aligment of these buttons to match with the figma design.

**Review by Andres2D — CHANGES_REQUESTED** [review]

File: pages/matchs/index.tsx (hunk 1 above)
Comment (urielPalomares) [reply pages/matchs/index.tsx]: Ready

File: pages/matchs/index.tsx (hunk 2 above)
Comment (urielPalomares) [reply pages/matchs/index.tsx:20]: Ready

File: pages/matchs/matchs.module.css (hunk 3 above)
Comment (urielPalomares) [reply pages/matchs/matchs.module.css]: Ready

File: pages/matchs/index.tsx (hunk 4 above)
Comment (urielPalomares) [reply pages/matchs/index.tsx]: Ready

File: pages/matchs/matchs.module.css (hunk 5 above)
Comment (urielPalomares) [reply pages/matchs/matchs.module.css:35]: Ready

File: pages/matchs/matchs.module.css (hunk 6 above)
Comment (urielPalomares) [reply pages/matchs/matchs.module.css]: Ready

**Review by Andres2D — APPROVED** [review]
Crack

**Pull request merged**